plugin-server/src/worker/vm/lazy.ts

```diff
@@ -249,9 +249,9 @@ export class LazyPluginVM {
     }
 
     private async processFatalVmSetupError(error: Error, isSystemError: boolean): Promise<void> {
-        this.hub.statsd?.increment('plugin_disabled_by_system', {
-            team_id: this.pluginConfig.team_id.toString(),
-            plugin_id: this.pluginConfig.plugin_id.toString(),
+        this.hub.statsd?.increment('plugin.disabled.by_system', {
```

Comment:
Annoyingly, out metrics system transforms all dots to underscores.

Reply:
I know, though in this case I'm totally okay with it, as the metric's name won't change